assets/js/sponsorshipLevelBenefit/edit.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+import $ from 'jquery';
+import Routing from '../Components/Routing';
+import '../../css/sponsorshipLevelBenefit/edit.scss';
+
+$(document).ready(function() {
+    updateLevelArrows();
+    updateBenefitArrows();
+    initSponsorshipLevelBenefitActions();
+});
+
+/* edit position of Sponsorhip Level */
+function sendLevelPosition(level_id, move)
+{
```

Comment:
Inconsistent location of curly braces, it's on the same line a few lines above, so should be the same here.